server/src/graphql.rs

```diff
@@ -57,6 +62,15 @@ impl MutationRoot {
 
 pub type Schema = RootNode<'static, QueryRoot, MutationRoot>;
 
-pub fn create_schema() -> Schema {
-    Schema::new(QueryRoot {}, MutationRoot {})
+pub fn make_schema() -> Schema {
+    RootNode::new(QueryRoot {}, MutationRoot {})
+}
+
+/// State shared across queries
```

Comment:
what kind of stuff goes in here? Do we want to add it now if its empty?

Reply:
e.g. database connection (in our setup it will be the ORM objects)

Added it here just to define the interface between app.rs and graphql.rs